### ISSUE TITLE
Flappy Bird Game

### ISSUE THREAD

**Issue by Sudhanshu069** [aditya-agr/Web-Dev-HacktoberFest-2K23#7]
I would like to add a js based game called Flappy Bird Game; kindly assign me the project

**Comment by aditya-agr** [comment]
Yes, Offcourse you can start by giving star and forking this repository.
